Automattic/jetpack
# Jetpack AI: fix image modal sizing issues

## Proposed changes:

Modal size changes to accommodate between image sizes and placeholder/blank images


https://github.com/user-attachments/assets/c9ef3c4a-3465-435e-8b27-f4637e0ddab1


This PR:

- changes the blank image placeholder for a lighter SVG
- force images to fit on the modal by resizing to 78% (considering all images are generated on 1024x768 pixels)

### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Open the image modal generation (general image or featured image). Be on the lookout for modal jumpiness/wonky behavior.

Generate an image, see when the blank/placeholder image turns into loading/generating state the modal doesn't change size.

See when the generated image is shown the modal doesn't change sizes.

Generate another image and check the same behavior.

Browse generated images, there should be no modal size changes while browsing.

Bonus: modal doesn't need to scroll to show entire content.
